Foundation/NSProcessInfo.swift

```diff
@@ -102,7 +102,19 @@ public class NSProcessInfo : NSObject {
     
     public var operatingSystemVersion: NSOperatingSystemVersion {
         get {
-            NSUnimplemented()
+            let productVersionKey = unsafeBitCast(_kCFSystemVersionProductVersionKey, UnsafePointer<Void>.self)
+            let productVersion = unsafeBitCast(CFDictionaryGetValue(_CFCopySystemVersionDictionary(), productVersionKey), NSString!.self)
+            if productVersion == nil {
+                fatalError("Could not read operating system version.")
```

Comment:
We shouldn't use a fatalError here because it will crash on Linux (where this dictionary does not exist).

Instead let's use a major/minor/patch version of 0. Even better, we should figure out what value we want to return on Linux and calculate it here. Kernel version maybe?